server/kit/kitserver_test.go

```diff
@@ -15,10 +15,59 @@ import (
 	ocontext "golang.org/x/net/context"
 	"google.golang.org/grpc"
 
+	gserver "github.com/NYTimes/gizmo/server"
 	"github.com/NYTimes/gizmo/server/kit"
 	"github.com/NYTimes/gziphandler"
 )
 
+func TestKitServerHTTPMiddleware(t *testing.T) {
+	shutdownErrChan := make(chan error)
+	go func() {
+		// runs the HTTP _AND_ gRPC servers
+		shutdownErrChan <- kit.Run(&server{})
+	}()
+
+	// wait for server to come up
+	time.Sleep(50 * time.Millisecond)
```

Comment:
Would it make more sense to put this in a retry loop and probe until the server is up? or are 50ms enough